test/test_tf_alb_listener_rules.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+import os
+from subprocess import check_call, check_output
+
+cwd = os.getcwd()
+
+
+class TestTFALBListenerRules(unittest.TestCase):
```

Comment:
I wouldn't like the underscores in the class name... The current name is the lesser of two weevils.